behave/features/steps/fala_end_to_end_tests.py

```diff
@@ -88,4 +102,42 @@ def step_impl_update_result_page(context, location, filter_label):
 def step_impl_error_shown_on_page(context):
     alert = context.helperfunc.find_by_css_selector(".alert-message")
     assert alert is not None
-    assert alert.text, "No results"
+    assert (
+        alert.text == "No results\nThere are no results matching your search criteria."
+    )
+
+
+@step(
+    'I am taken to the page corresponding to the "{location}" "{organisation}" search result'
+)
+def step_impl_result_page_with_multi_params(context, location, organisation):
+    result_number_paragraph = context.helperfunc.find_by_xpath(
+        '//p[@class="govuk-body"]'
+    )
+    current_url = context.helperfunc.driver().current_url
+    title_xpath = context.helperfunc.find_by_xpath(
+        "//html/body/div/main/div/div/h1"
+    ).text.replace("\n", " ")
+    result_container_xpath = context.helperfunc.find_by_xpath(
+        '//div[@class="search-results-container"]'
+    )
+    organisation = organisation.replace(" ", "+")
+    location = location.replace(" ", "+")
+
+    assert (
+        current_url
+        == f"""{CLA_FALA_URL}/?postcode={location}&name={organisation}&search="""
+    )
+    assert title_xpath == f"{FALA_HEADER}"
+    assert result_container_xpath, result_number_paragraph is not None
+
+
+@step('"{count}" result is visible on the results page')
```

Comment:
We can use Pre-defined Data types https://www.tutorialspoint.com/behave/behave_data_types.htm

So it would be `{count:d}` That accepts a digital which in this case is a number. No need to convert a String to an integer. It's something we don't take advantage of much.

Reply:
Added